parts/provision.sh

```diff
@@ -30,11 +30,11 @@ function setup_ubuntu() {
 
   case $version in
     "18.04")
-      sgx_driver_url="https://download.01.org/intel-sgx/dcap-1.2/linux/dcap_installers/ubuntuServer18.04/sgx_linux_x64_driver_1.12_c110012.bin"
+      sgx_driver_url="https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntuServer18.04/sgx_linux_x64_driver_1.21.bin"
```

Comment:
This is probably flawed as the driver name will probably change for every release. The latest directory is definitely a good idea, but the naming scheme is flawed and we cannot tell what the name of the driver is.
we will need to lock into the actual versioned version instead or we will get broken next time Intel releases.